code/game/turfs/open/floor/plating/misc_plating.dm

```diff
@@ -131,6 +151,147 @@
 	icon_state = "sand"
 	baseturfs = /turf/open/floor/plating/beach/sand
 
+/turf/open/floor/plating/beach/water
+	gender = PLURAL
+	name = "water"
+	desc = "You get the feeling that nobody's bothered to actually make this water functional..."
```

Comment:
I dislike the copy paste and it would be better off as a component, but I think I am willing to let it slide. This description should probably be updated